src/Compilers/CSharp/Portable/Syntax/AnonymousMethodExpressionSyntax.cs

```diff
@@ -15,6 +15,16 @@ public AnonymousMethodExpressionSyntax Update(SyntaxToken asyncKeyword, SyntaxTo
             => body is BlockSyntax block
                 ? Update(asyncKeyword, delegateKeyword, parameterList, block, null)
                 : Update(asyncKeyword, delegateKeyword, parameterList, null, (ExpressionSyntax)body);
+
+        public override SyntaxToken AsyncKeyword
+            => this.Modifiers.FirstOrDefault(SyntaxKind.AsyncKeyword);
+
+        internal override AnonymousFunctionExpressionSyntax WithAsyncKeywordCore(SyntaxToken asyncKeyword) => WithAsyncKeyword(asyncKeyword);
```

Comment:
>=> [](start = 107, length = 2)

nit: line break before `=>` and also before next member?

Reply:
These were actually copied out of the generated code.  But sure, i don't mind tweaking this :)